docs/4.4/preview/teleport-application-access.md

```diff
@@ -0,0 +1,207 @@
+# Teleport Application Access Preview
+
+Teleport is currently beta testing an application access proxy. Teleport Application Access has been designed to secure internal web applications, letting you provide secure access while improving both visibility and control for access.
+
+Here are a few things you might want to secure with Teleport Application Access:
+
+- Internal Control Panels.
+- Wikis / Tooling that's only available on the VPN.
+- Access to the Kubernetes Dashboard.
+- Developer tools. Such as Jenkins, or the Atlassian stack.
+
+**Example Teleport Application Usage**
+
+This diagram shows Teleport deployed into an AWS VPC. Teleport Application Access is now providing access to Jenkins and an internal dashboard. Another Teleport Application Service is running in another datacenter and dials back to the Teleport cluster. This enables users to access that other dashboard.
+![Example App Access Usage](../img/aap.svg)
+
+_Want to see a demo of AAP in action? [Click here](#view-applications-in-teleport)_
```

Comment:
Does this link go anywhere? (it doesn't seem to)

Reply:
Removed this note / link